src/main/java/org/ow2/proactive/addons/ldap_query/LDAPClient.java

```diff
@@ -98,10 +99,10 @@ public LDAPClient(Map<String, Serializable> actualTaskVariables, Map<String, Ser
 
         for (String variableName : taskVariablesList) {
```

Comment:
the whole for could be refactored and cleaned using lambdas. Also the mapWithVariables is not used outside the for loop, so there is not really need for it.